oakcan-up.sh

```
@@ -0,0 +1,83 @@
+  # Start all infra services
+  for infra_service in $infra_services; do
+    echo "Starting $infra_service..."
+    docker-compose -f docker-compose.infra.yml up -d "$infra_service"
+  done
+}
+
+# Function to rebuild and restart a specific service
+rebuild_service() {
+  local service=$1
+
+  if [[ " ${services[*]} " =~ "${service}" ]]; then
```

Shellcheck (0.10.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)